package.json

```diff
@@ -3,13 +3,22 @@
   "version": "0.3.3",
   "description": "All dreipol eslint and prettier rules to build any kind of project",
   "main": "index.js",
+  "engines": {
+    "npm": ">=6",
+    "node": ">=12"
+  },
   "scripts": {
-    "test": "node -e \"require('./')\""
+    "check-format": "prettier --check \"./**/*.js\"",
+    "lint": "eslint ./",
+    "all-tests": "node tests/run.js",
```

Comment:
can't we start the tests with `tape tests/**/*.test.js`? Instead of loading them from a single file?